app/views/question/_question.haml

```diff
@@ -4,7 +4,7 @@
       .media
         - unless question.author_is_anonymous
           %a.pull-left{ href: unless hidden then user_path(question.user) end }
-            %img.answerbox__question-user-avatar.avatar-md{ src: question.user.profile_picture.url(:medium) }
+            %img.answerbox__question-user-avatar.avatar-md{ src: question.user.profile_picture.url(:small) }
```

Comment:
This one is missing a lazy load. Is this intentional?

Reply:
Yes, because it's always above the fold, so it doesn't require to be lazy-loaded.